main/_config.py

```diff
@@ -11,6 +11,12 @@ class Config(BaseSettings):
     SQLALCHEMY_DATABASE_URI: str
     SQLALCHEMY_ENGINE_OPTIONS: dict = {}
     SQLALCHEMY_ECHO: bool = False
+    MYSQL_DEV_ROOT_PASSWORD: str
+    MYSQL_DEV_DATABASE: str
+    MYSQL_DEV_PORT: int = 3306
+    ADMINER_PORT: int = 8080
```

Comment:
Why do we need these configs?

Reply:
It's used in docker-compose file

Reply:
We should only define the configs directly used by the application. Let's remove them if they are used only in docker compose.